### PR TITLE
DGR-464-BE-Add the additional attributes to Apiary

### PR DESCRIPTION
**Pull request by shclarke007**
Updates all Group Public API endpoints to have the additional attributes documented:

Attribute | Date type 
-- | --
auto_expiry | integer (year) 
signup_url_show | boolean
signup_url | text
course_link_show | boolean 
organization_link_show | boolean

Wording for attributes description to be confirmed by Kelly 

`primary_design_id` added as was in the response of the API but not documented

### CODE REVIEW

File: apiary.apib
Comment (albertpaulp) [inline apiary.apib:1459]: Just wanted to double check, have you run these texts with Kelly?

Comment (shclarke007) [reply apiary.apib:1459]: Yes, I'm waiting for confirmation before merging

File: apiary.apib
Comment (kei178) [inline apiary.apib:1485]: I find this misleading. "auto_expiry" should not be `2024` but `1` or `2` as in 1 year later or 2 years later.

File: apiary.apib
Comment (kei178) [inline apiary.apib:1463]: There are syntax errors. `-` is not necessary between the filed name and `(type, required)`. Because of this, the data type is displayed incorrectly.

Screenshot 2024-06-11 at 11 18 45

**Review by kei178 — CHANGES_REQUESTED** [review]

File: apiary.apib
Comment (shclarke007) [reply apiary.apib:1463]: good catch

**Review by kei178 — APPROVED** [review]

**Pull request merged**
